src/app/visual/repeater/repeater-visual.component.ts

```diff
@@ -109,4 +109,12 @@ export class RepeaterVisualComponent {
   public onItemClick(index: number): void {
     this.activeIndex = index;
   }
+
+  public onEnter(event: KeyboardEvent, index: number): void {
+    this.onItemClick(index);
+  }
+
+  public onSpace(event: KeyboardEvent, index: number): void {
```

Comment:
Should we make the click and keyboard interactions a baked-in feature of the repeater if the repeater component has set the `activeIndex` property (or the `selectable` property on the items)? This feels like "busy work" we're offloading on the consumer that will more or less be duplicated for every repeater that wants an active state.

Reply:
We certainly could, but first here's my two concerns:
1. Would it be weird to offer an `activeIndex` input but also set that for them when the user clicks/navigates?
2. Would this be considered a breaking change? They would have to change their code to no longer update `activeIndex` on click, since we will be handling it for them.

Reply:
I suppose it would be a breaking change unless we created some way to opt-in to the behavior (via a new input or something).

It just feels odd that we could have some implementations of repeater that activate _only_ when the user clicks the item, or some that activate when the user presses return (and not the space bar), etc. because the interaction is left up to the consumer. These kinds of interactions should be exactly the same for all repeaters across all of our SPAs.

Reply:
Agree that the interactions need to be the same across implementations, however we get there (building in or providing good code templates)

Reply:
Sidebar > we decided to make the `activeIndex` property support 2-way binding. Working on that now...